projects/plugins/backup/readme.txt

```diff
@@ -14,6 +14,8 @@ VaultPress is the most proven WordPress backup plugin with over 270 million back
 
 Jetpack VaultPress Backup is a plugin that gives you cloud-based backups and one-click restores without needing to install the full Jetpack plugin.
 
+VaultPress is the most proven WordPress backup plugin with over 270 million backups. Save each change and get back online fast with one-click restores.
```

Comment:
Noting there was discussion in Slack about this line as well as the line below it was copied from. p1682524957608919/1682100655.060309-slack-C01264051NE